content/docs/ui/sending-email/getting-started-with-automation.md

```diff
@@ -168,3 +168,16 @@ _To delete an automation:_
 1. On the automation dashboard, find the automation you want to delete.
 1. Hover over the action menu and select the delete icon.
 1. To delete the automation, click **Confirm**.
+
+## Automation Use Cases and Examples
+
+### Segment-triggered Automation
+
+With this functionality, new contacts can flow into and out of Automations more… automatically! For example, if you have a segment of “Free plan customers” who you're nurturing to become paid customers, you can do the following:  
+
+1. Create a segment of all contacts whose custom field “Plan Type” value is “Free”
+1. Set entry criteria to “first time a contact is added to” the “Free plan customers” segment
```

Comment:
Are we capitalizing Entry Criteria / Exit Criteria? Wasn't sure what's best, but want to be consistent. 
For row 179, missing "Set the entry criteria" and suggest simply "Free customers"

Reply:
We shouldn't be if it's not capitalized in the app. I'll do a find/replace to make sure we don't have any rogue capitalization in the doc.